### DISCUSSION TITLE
Website instead of Wiki?

### DISCUSSION THREAD

**Discussion by ElMassimo** [windicss/windicss#21] [Ideas]
In some cases, migrating from TailwindCSS will be straightforward.

In other cases, guidance will be necessary.

At the moment, scanning through the Wiki might not be too friendly for folks that are new to windicss.

How would you feel about adding a small website to the repo using Vitepress? @voorjaar @antfu 

Happy to get it started, and fill in the initial sections with the content we already have in the wiki.

#### Edit

Docs now have their own [repository](https://github.com/windicss/docs) and the [website is live](https://windicss.netlify.app/)!

**Comment by antfu** [comment]
Agree that Vitepress can be a good starting point. @voorjaar not sure if you'd like to use Svelte or something else. Otherwise, I think @ElMassimo and me can handle the setup if you are not familiar with it.

**Reply by voorjaar** [reply]
Yes, I have never used Vitepress, but I think I will get started. It would be great if you can handle the setup, and I will help you later.

**Reply by antfu** [reply]
@ElMassimo Would you like to kickstart it? Or I can do it later this week.

**Reply by antfu** [reply]
In case you don't know, the wiki can be cloned as markdowns so you don't need to grab them manually 
`git clone https://github.com/windicss/windicss.wiki.git`

**Reply by ElMassimo** [reply]
Awesome, I'll get started on a `docs` branch. Thanks for the tip about the wiki, didn't know!

**Reply by antfu** [reply]
Maybe it's better in a separate repo for the docs site? WDYT @voorjaar?

**Comment by ElMassimo** [comment]
Leaving the logo for the other discussion, used a plain icon for now. This is what I've got so far:

<img width="1428" alt="Screen Shot 2021-02-15 at 17 31 04" src="https://user-images.githubusercontent.com/1158253/107991581-aef53000-6fb5-11eb-8ab3-39a7d7abc1b5.png">

**Reply by ElMassimo** [reply]
Took some liberties with the copy for the features section, to make it friendlier for newcomers.

<img width="1425" alt="Screen Shot 2021-02-15 at 17 47 19" src="https://user-images.githubusercontent.com/1158253/107991667-dea43800-6fb5-11eb-9688-026afe84766f.png">

For example, I renamed "Unrestricted Build" to "Auto-Inferred Utilities".

**Reply by antfu** [reply]
Looks awesome! I think it's a good placeholder for now and we can figure out the logo later.

**Reply by voorjaar** [reply]
Wow!!!  So cool. Maybe little shadow on cards.

**Reply by ElMassimo** [reply]
Will make a PR now, that way we can split the work of different pages if needed.

**Reply by ElMassimo** [reply]
https://github.com/windicss/windicss/pull/25

**Comment by ElMassimo** [comment]
Website is now [live](https://windicss.netlify.app/)

**Reply by voorjaar** [reply]
Really amazing~
